tiled/catalog/orm.py

```diff
@@ -95,6 +96,12 @@ class Node(Timestamped, Base):
             "metadata",
             postgresql_using="gin",
         ),
+        # This index supports full-text search.
+        Index(
+            "metadata_search",
```

Comment:
I think it's worth giving this a pretty descriptive name, since we anticipate adding others in the future. I could see potentially adding other types of text indexes, like trigram etc. and it will help if the names are unambigious.

I'm not quite sure how far to take it. Maybe `metadata_tsvector`? Or `metadata_tsvector_case_insensitive`?

Reply:
This is running against SQLite engines as well and failing a slew of tests. Needs to be made conditional on dialect somehow.